### PR TITLE
Add switch for disabling proxy get_url fetches

### PR DESCRIPTION
**Pull request by yunghuffy**
Working from https://github.com/geerlingguy/ansible-role-jenkins/pull/120 this change aims to modify the behavior of all `get_url` fetches so that the proxy behavior can optionally be enabled or disabled.

### CODE REVIEW

**Comment by acrewdson** [comment]
+1, would be great to have this change; based on what you said [here](https://github.com/geerlingguy/ansible-role-jenkins/pull/120#discussion_r114344675]), @geerlingguy, you'd be up for merging this?

File: README.md [hunk 1]
```diff
@@ -109,6 +109,11 @@ Extra Java options for the Jenkins launch command configured in the init file ca
 
 Changes made to the Jenkins init script; the default set of changes set the configured URL prefix and add in configured Java options for Jenkins' startup. You can add other option/value pairs if you need to set other options for the Jenkins init file.
 
+    jenkins_use_proxy: yes
+
+Whether or not to use the system supplied proxy when fetching information via
+`get_url`.
```
Comment (geerlingguy) [inline README.md]: This should be on one line (for markdown files, I don't break at 80 cols).

**Comment by geerlingguy** [comment]
With the exception of the review comment, I'm happy to merge this in, thanks!

File: README.md (hunk 1 above)
Comment (geerlingguy) [reply README.md]: Markdown files are generally rendered in a UI or preview tool where it's better to let your editor/GitHub/etc. control the text flow, so I prefer for files which are more end-user-facing to have freeform paragraphs (which are also easier to edit in long form) than README files which might be intended only for use in editors or on the CLI.

Comment (acrewdson) [reply README.md]: @geerlingguy thanks for that explanation 👍 for me wide line lengths are going to be a problem in any file where you might want to view a side-by-side diff (like what I am looking at here on github.com), regardless of the type

**Comment by champain** [comment]
Testing failures seem unrelated and the results are very confusing. This appears to be unrelated to my requested change.

**Comment by champain** [comment]
@geerlingguy any progress here?

**Comment by imuchnik** [comment]
@geerlingguy Any updates on whether or not this will be merged?

**Pull request closed without merging**